src/website/src/pages/eu/about/digital-design-principles/index.mdx

```diff
@@ -14,31 +14,50 @@ import { Paragraph } from '@ecl/website-components';
   findings.
 </Paragraph>
 
+<div style={{borderLeft: '4px solid #0E47CB', marginTop: '3rem', paddingLeft: '1rem'}}>
+
 ## Connecting with people
```

Comment:
These h2s they get the border bottom that i don't see in the specs but the border is in place in the mocks for the "standalone" h2s, apparently... These are also black in the mock, while the others are grey. I don't know if we are supposed to reproduce all of these design specs or inconsistencies, depending on the way we look at them..